gadgets/fsnotify/gadget.yaml

```diff
@@ -0,0 +1,117 @@
+name: fsnotify
+description: Gadget enriching inotify or fanotify events
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://inspektor-gadget.io/docs
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+datasources:
+  fsnotify:
+    fields:
+      fa_f_flags:
+        annotations:
+          description: event_f_flags from fanotify_init
+      fa_flags:
+        annotations:
+          description: flags from fanotify_init
+      fa_mask:
+        annotations:
+          description: Mask describing fanotify event (FAN_ACCESS, FAN_MODIFY, etc.)
+      fa_pid:
+        annotations:
+          description: pid from fanotify_event_metadata. Process ID, or thread ID
+            if flag FAN_REPORT_TID was set in fanotify_init
+      fa_response:
+        annotations:
+          description: 'response to a fanotify permission request: allow or deny'
```

Comment:
nit: Why `'` are used in some cases and no in others?

Reply:
When the string contains a `:`, it needs to be quoted, otherwise the yaml parser gets confused.
I think that `--update-metadata` removes the quotes when not needed.
About `'` vs `"`, I don't know.

Reply:
You got a point!, I didn't realize there were too many `:` in this file.